stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/style/StylesHolder.kt

```diff
@@ -0,0 +1,7 @@
+package io.getstream.chat.android.ui.style
+
+import io.getstream.chat.android.ui.message.list.MessageListViewStyle
+
+internal object StylesHolder {
+    var messageListViewStyle: MessageListViewStyle? = null
```

Comment:
Can't we have a non-nullable property here?

Reply:
🤔  The way I see it to use `lateinit var`. What do your think?

Reply:
Done with static values